examples/FileScan/Scan_RequestUploadURL_Post/Program.cs

```diff
@@ -41,7 +41,7 @@ static async Task Main(string[] args)
             string url = urlResponse.Result.PostURL ?? "undefined url";    // This case should never happen
 
             // Create uploader and upload file
-            var uploader = new FileUploader.Builder().Build();
+            var uploader = new PangeaCyber.Net.FileScan.FileUploader.Builder().Build();
```

Comment:
Why do we have 2 `FileUploader`s anyways? They look like they do the same thing.

Reply:
We should have deprecated FileScan.FileUploader in last mayor version. Originally it was done there because it was only used for FileScan, Share and Sanitize were not even planned, but now they could share this same FileUploader. 
I've updated the example to use  `PangeaCyber.Net.FileUploader` now.

Reply:
https://github.com/pangeacyber/pangea-dotnet/pull/475